packages/venia-ui/lib/components/SearchPage/searchPage.js

```diff
@@ -12,16 +12,30 @@ import defaultClasses from './searchPage.css';
 import PRODUCT_SEARCH from '../../queries/productSearch.graphql';
 import FILTER_INTROSPECTION from '../../queries/introspection/filterIntrospectionQuery.graphql';
 import GET_PRODUCT_FILTERS_BY_SEARCH from '../../queries/getProductFiltersBySearch.graphql';
+import CategorySort from '../CategorySort';
 
 const SearchPage = props => {
     const classes = mergeClasses(defaultClasses, props.classes);
 
+    const [sort, setSort] = useState({
+        sortAttribute: 'relevance',
+        sortDirection: 'ASC'
+    });
+
+    const { sortAttribute, sortDirection } = sort;
+
+    const sortControl = {
+        currentSort: sort,
+        setSort: setSort
+    };
```

Comment:
All of this logic, and the label logic below should all be contained within the `useSearchPage` talon. That should make it a little cleaner when determining the label; you could use two Maps and concat labels, or a single Map with hashed keys that are a combination of attribute/direction.